tests/v2/validation/provisioning/rke2/custom_cluster_test.go

```diff
@@ -97,6 +98,8 @@ func (c *CustomClusterProvisioningTestSuite) TestProvisioningRKE2CustomCluster()
 		{"2 nodes - etcd/cp roles per 1 node " + provisioninginput.StandardClientName.String(), c.standardUserClient, nodeRolesShared, false, c.client.Flags.GetValue(environmentflag.Short) || c.client.Flags.GetValue(environmentflag.Long)},
 		{"3 nodes - 1 role per node " + provisioninginput.AdminClientName.String(), c.client, nodeRolesDedicated, false, c.client.Flags.GetValue(environmentflag.Long)},
 		{"3 nodes - 1 role per node " + provisioninginput.StandardClientName.String(), c.standardUserClient, nodeRolesDedicated, false, c.client.Flags.GetValue(environmentflag.Long)},
+		{"4 nodes - etcd/cp roles per 1 node + 1 windows worker" + provisioninginput.AdminClientName.String(), c.client, nodeRolesSharedWindows, true, c.client.Flags.GetValue(environmentflag.Long)},
```

Comment:
we should test this with dedicated nodes - its the most common use case for windows clusters - instead of a stacked node setup.

Reply:
maybe just call it `1 role per node, with windows` or something?

Reply:
But we already have the dedicated nodes, this was just adding stacked nodes to that existing usecase.

Reply:
Updated after offline discussion